bin/start_lexical.sh

```diff
@@ -35,16 +35,10 @@ esac
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-PATH_APPEND_ARGS=$(for f in $(find ${SCRIPT_DIR}/../lib -name '*.ez')
-do
-    lib=$(basename $f | sed -e 's/.ez//g')
-    echo "-pa $f/$lib/ebin"
-done)
-
-"${ELIXIR_COMMAND}" $(echo $PATH_APPEND_ARGS) \
-                    -pa "${SCRIPT_DIR}/../consolidated" \
-                    -pa "${SCRIPT_DIR}/../config/" \
-                    -pa "${SCRIPT_DIR}/../priv/" \
-                    --eval "LXical.Server.Boot.start" \
-                    --cookie "lexical" \
-                    --no-halt
+"${ELIXIR_COMMAND}" \
+    -pa "${SCRIPT_DIR}/../consolidated" \
+    -pa "${SCRIPT_DIR}/../config/" \
+    -pa "${SCRIPT_DIR}/../priv/" \
```

Comment:
do we still need to set these via `-pa` or can we use `:code.add_patha?`?

Reply:
looks like you already answered that in the comment. Let's add them in the script then.